tests/manage/z_cluster/test_must_gather.py

```diff
@@ -45,3 +49,26 @@ def test_must_gather(self, mustgather, log_type):
         """
         mustgather.log_type = log_type
         mustgather.validate_must_gather()
+
```

Comment:
`tier4` marker is missing

Reply:
@ebenahar 
need to add `tier4` and `tier4a` on same test?